apis/fluent-en_GB/atrium-api-fluent-en_GB/src/jvmTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/DateSubjectChangerSamples.kt

```diff
@@ -44,7 +45,45 @@ class DateSubjectChangerSamples {
             expect(date).asLocalDate {
                 toBeAfter(LocalDate.parse("2025-07-17"))            // fails
                 toBeBefore(LocalDate.parse("1994-07-17"))           // still evaluated even though `toBeAfter` already fails
-                                                                    // use `.asLocalDate().` if you want a fail fast behaviour
+                                                                        //  use `.asLocalDate().` if you want a fail fast behaviour
+            }
+        }
+    }
+
+    @Test
+    fun asLocalDateTimeFeature() {
+        val date = formatter.parse("1995-07-17")
+
+        expect(date)
+            .asLocalDateTime() // subject is now of type LocalDate
+            .toEqual(LocalDateTime.parse("1995-07-17T00:00:00"))
+
+        fails {
+            expect(date)
+                .asLocalDateTime()                                  // subject is now of type LocalDateTime
+                .toBeAfter(LocalDateTime.parse("2025-07-17T00:00:00"))       // fails
+                .toBeBefore(LocalDateTime.parse("1996-07-17T00:00:00"))      // not evaluated/reported because `toBeAfter` already fails
+                                                                                  // use `.asLocalDateTime { ... }` if you want all assertions evaluated
```

Comment:
maybe only a problem of github, the alignment looks wrong.

Reply:
I don't know but in my environment looks good but when I see the changes in the commit tool in IntelliJ the alignment looks wrong. Do you mind going into my branch and checking in your environment @robstoll ?